Yosemite/Yosemite/Stores/ShippingLabelStore.swift

```diff
@@ -247,14 +253,13 @@ private extension ShippingLabelStore {
                     labelPurchaseIDs.append(label.shippingLabelID)
                 }
 
-                // Wait for 2 seconds (give the backend time to process the purchase)
-                DispatchQueue.main.asyncAfter(deadline: .now() + 2.0) { [weak self] in
+                // Wait to give the backend time to process the purchase
+                DispatchQueue.main.asyncAfter(deadline: .now() + backendProcessingDelay) { [weak self] in
```

Comment:
I wonder whether it's appropriate to run this on the main queue or it would be better to use a background queue and trust the UI layer calling this to update on the main thread.... Out of scope.